### PR TITLE
Added rule to post TSC reminder to slack

### PR DESCRIPTION
**Pull request by nmaludy**
**Changes**
- Creates a reminder for those of us in the TSC about the upcoming TSC meeting.
  - This rule should fire 30 minutes prior to the meeting.
- Added CircleCI config, somehow this was missing. I setup CircleCI on my fork and here are the results: https://app.circleci.com/jobs/github/nmaludy/stackstorm-st2community/4
- Added CHANGELOG.md

### CODE REVIEW

**Comment by nmaludy** [comment]
Looks like the Python 2 CI is still busted. I'll see if i can look into that today.

**Comment by nmaludy** [comment]
After clearing the CI cache the build passed just fine: https://circleci.com/gh/nmaludy/stackstorm-st2community/tree/feature%2Ftsc-notification

**Review by arm4b — COMMENTED** [review]
Nice! :+1:  Additional thanks for the tidy up work for this pack, including license & changelog.

File: rules/post_tsc_reminder_to_slack.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+name: "post_tsc_reminder_to_slack"
+pack: "st2community"
+description: "Post a reminder about the TSC meeting to Slack."
+enabled: true
+
+trigger:
+  type: "core.st2.CronTimer"
+  parameters:
+    timezone: "UTC"
+    day: 1st tue
```
Comment (arm4b) [inline rules/post_tsc_reminder_to_slack.yaml:11]: @nmaludy Just to make sure, did you double check if this kind of format works & recognized?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,144 @@
+---
+version: 2
```
Comment (arm4b) [inline .circleci/config.yml:2]: With CircleCI build added, makes sense to also include the build status badge to the README.

File: rules/post_tsc_reminder_to_slack.yaml (hunk 1 above)
Comment (nmaludy) [reply rules/post_tsc_reminder_to_slack.yaml:11]: I didn't check, no although it is documented and i looked at the code for the scheduler library.

**Review by arm4b — APPROVED** [review]

**Comment by arm4b** [comment]
Just deployed it to the current st2cicd server.

**Pull request merged**